manifests/init.pp

```diff
@@ -119,7 +119,10 @@
       icvpn_as => $mesh_as;
     }
   }
- 
+  
+  ffnord::resources::ffnord::field {
+    "MESH_CODE": value => "${mesh_code}";
```

Comment:
So this should be an array? and then we would have to iterate over that array in the check script?